libraries/abstractions/common_io/test/test_iot_spi.c

```diff
@@ -311,7 +311,7 @@ TEST( TEST_IOT_SPI, AFQP_IotSPI_ReadSyncAssisted )
 
         lRetVal = iot_spi_ioctl( xSPIHandle, eSPIGetRxNoOfbytes, &xBytesRx );
         TEST_ASSERT_EQUAL( IOT_SPI_SUCCESS, lRetVal );
-        TEST_ASSERT_EQUAL( xBytesRx, 4 );
+        TEST_ASSERT_EQUAL( xBytesRx, sizeof( ucRxBuf ) );
```

Comment:
It looks to me like this changes an assertion that the variable is 4 to an assertion that the variable is 16.  Unless I'm reading this wrong, the assert either had to be failing before, or must fail with this change.  Have you run this and seen all test cases passing?

Reply:
Yes. It is verified. It was commented in the internal branch because the api was not ready. So the wrong count of bytes was found out in the first check in.